### ISSUE TITLE
lack of native debugging tools

### ISSUE THREAD

**Issue by JeriXNet** [Kode/Kha#102]
I've noticed(tried on windows only for now) that haxe.CallStack.callStack() array is always empty. It can be really useful sometimes. I understand that it must be disabled by default but may be it should be enabled with some flag to khamake like `-debug`?

**Comment by JeriXNet** [comment]
I think I've found the problem:
I've added ```HXCPP_DEBUG;``` to ```PreprocessorDefinitions``` tag directly in generated vcxproj and it worked ok!

**Comment by JeriXNet** [comment]
I've tried to work some time with this define enabled and had difficulties to catch null reference errors. I mostly work in FD and been able to catch this type of errors with this define but callstack was still emty. Furhermore, when I launched debug in msvs to catch error, this ide wasn't stop on it. I removed this define and then msvs succesfully showed exact place in c++ code. So, I think problem may be hidden somewhere deep in haxe or hxcpp and adding this define in debug conf for msvs isn't such a good idea after all.
So now I'm working with Kha with all defaults in FD and sometimes launch msvs.
But I don't want to close this issue for now...
